IdentityCore/src/validation/MSIDAuthority.m

```diff
@@ -253,6 +253,8 @@ + (BOOL)isAuthorityFormatValid:(NSURL *)url
     return YES;
 }
 
+- (void)replaceURL:(NSURL *)url { _url = url; }
```

Comment:
why do you need this? There're a lot of things being set up in the initializer, so I'd be extremely cautious about method like this (e.g. tenant, authority format validation etc are all done in initializer)

Reply:
forgot to remove this, not it's gone!